tools/Azure.Mcp.Tools.Monitor/src/Commands/HealthModels/Entity/EntityGetHealthCommand.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Azure.Mcp.Tools.Monitor.Commands.HealthModels.Entity;
 
-public sealed class EntityGetHealthCommand(ILogger<EntityGetHealthCommand> logger) : BaseMonitorHealthModelsCommand<EntityGetHealthOptions>
+public sealed class EntityGetHealthCommand(ILogger<EntityGetHealthCommand> logger) : BaseMonitorHealthModelsCommand<BaseMonitorHealthModelsOptions>
```

Comment:
Not this change, but related, what are your thoughts on cases where we have

`BaseOptions` with properties X, Y, and Z
`Command1Options : BaseOptions`, no properties added
`Command2Options : BaseOptions`, adds property W

Do we want to have `Command1Options` defined given it doesn't add anything?